packages/shared/src/components/modals/post/CreateSharedPostModal.tsx

```diff
@@ -72,41 +71,44 @@ export function CreateSharedPostModal({
     checkUrl(value);
   };
 
+  const footer = (
+    <>
+      <Button
+        icon={<AtIcon />}
+        className="btn-tertiary"
```

Comment:
👀  this should rather be a buttonVariant?

Reply:
Yep, let me make it modern-looking.